src/redux/mission/MissionsSlice.js

```diff
@@ -0,0 +1,62 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import axios from 'axios';
```

Comment:
- [ ] same here, Please remove this eslint disablement line and fix linters issues yourself. Linters is a tool to help you write clean and error-free code so please don't disable it and try to fix it's errors.